dist/api_data.py

```diff
@@ -771,6 +771,9 @@ def __cmp__(self, other):
     Config('remove_files', 'true', r'''
         should the underlying files be removed?''',
         type='boolean'),
+    Config('lock_wait', 'true', r'''
+        wait for locks, if false will return \c EBUSY instead of blocking''',
```

Comment:
This implies that it won't return EBUSY if you don't specify ``lock_wait``, I don't think that's true (just much less likely)